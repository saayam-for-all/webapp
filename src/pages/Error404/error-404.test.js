import { render } from "@testing-library/react";
import Error404 from "./Error404";

describe("Error404", () => {
  it("renders correctly", () => {
    const tree = render(<Error404 />);
    expect(tree).toMatchSnapshot();
  });
});
