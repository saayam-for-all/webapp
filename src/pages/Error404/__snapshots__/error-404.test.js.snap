// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error404 renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <section
          class="page_404"
        >
          <div
            class="container"
          >
            <div
              class="row"
            >
              <div
                class="col-sm-12 "
              >
                <div
                  class="col-sm-10 col-sm-offset-1  text-center"
                >
                  <div
                    class="four_zero_four_bg"
                  >
                    <h1
                      class="text-center "
                    >
                      404
                    </h1>
                  </div>
                  <div
                    class="contant_box_404"
                  >
                    <h3
                      class="h2"
                    >
                      Look like you are lost the page you are looking for not available!
                    </h3>
                    <mock-link
                      props="{"className":"link_404"}"
                      to="/"
                    >
                      Go to Home
                    </mock-link>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </section>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <section
        class="page_404"
      >
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="col-sm-12 "
            >
              <div
                class="col-sm-10 col-sm-offset-1  text-center"
              >
                <div
                  class="four_zero_four_bg"
                >
                  <h1
                    class="text-center "
                  >
                    404
                  </h1>
                </div>
                <div
                  class="contant_box_404"
                >
                  <h3
                    class="h2"
                  >
                    Look like you are lost the page you are looking for not available!
                  </h3>
                  <mock-link
                    props="{"className":"link_404"}"
                    to="/"
                  >
                    Go to Home
                  </mock-link>
                </div>
              </div>
            </div>
          </div>
        </div>
      </section>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
